src/test/kotlin/calculator/StringCalculatorTest.kt

```diff
@@ -0,0 +1,19 @@
+package calculator
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class StringCalculatorTest : StringSpec({
+    "문자열을 입력 받아 사칙 연산을 수행한다." {
+        val stringCalculator = StringCalculator(ExpressionFactory(OneSpaceSeparationStrategy()))
+        val parameters = listOf(
+            StringCalculatorFixture("2 + 3 * 4 / 2", 10),
+            StringCalculatorFixture("2 * 8 / 3 - 5", 0),
+            StringCalculatorFixture("3 + 4 - 10 * 8", -24),
+        )
+
+        parameters.forEach {
+            stringCalculator.calculate(it.expression) shouldBe it.expected
+        }
```

Comment:
테스트하기 쉽도록 CalculatorFixtures 파일에 wrapping 클래스를 만들어 주셨네요 👍 
테스트 코드도 결국 코드이기 때문에, 테스트케이스가 추가될 떄마다 매번 data class가 생겨나게 되고, 결국 코드 관리가 번거로워질 것 같아요.
그리고 StringCalculatorFixture 객체를 생성하는 코드를 보았을 때, 이 것이 무엇을 의미하는지 잘 모르겠네요. 😭 

테스트코드가 로직을 검증하는 역할도 하지만, 그 과정도 중요합니다.
Wrapping 클래스를 사용하지 않고 테스트 코드를 작성해보면 어떨까요? (이하 관련 내용 동일)

```suggestion
        val expressionGroup = listOf(
            "2 + 3 * 4 / 2",
            "2 * 8 / 3 - 5",
            "3 + 4 - 10 * 8"
        )
        val expectedGroup = listOf(10, 0, -24)

        expressionGroup.forEachIndexed { index, expression ->
            stringCalculator.calculate(expression) shouldBe expectedGroup[index]
        }
```
혹시나 해서 예시를 적어두었습니다 😅

Reply:
각각 나누어서 변수에 할당하는 것도 좋은 방식이네요! 👍 
다만 생각이 드는 것이 Junit으로 구현하게 된다면 @CsvSource로 간단하게 구현가능할 것 같다는 생각도 드는데..
이건 제가 아직 Kotest를 제대로 활용하고 있지 않은 문제인 것 같네요 😂